### PR TITLE
Console report

### LINKED ISSUE #37: Report console output
The report from forklift when printed to the console is a bunch of html and is not helpful. We should do something easier to read and still color coded. The nose-cover output would be a good example.

### PR DESCRIPTION
**Pull request by steveoh**
closes #37 

:art: :sparkles:

### CODE REVIEW

File: src/forklift/cli.py [hunk 1]
```diff
@@ -133,46 +136,66 @@ def start_lift(file_path=None):
...
 
     pallet_reports = lift.process_pallets(pallets)
+    report_object = lift.create_report_object(pallet_reports, elapsed_time)
 
-    _send_report_email(pallet_reports)
+    email = get_config_prop('sendEmails')
+    if email:
+        _send_report_email(report_object)
 
+    print('Finished in %s. General email notification: %s', elapsed_time, email)
```
Comment (stdavis) [inline src/forklift/cli.py]: Should this be a `log.info` statement so that it goes in the log file?

Comment (steveoh) [inline src/forklift/cli.py]: nah i dont' think so, it's for the console. elapsed time is log.info'd it on line 140.

Comment (stdavis) [inline src/forklift/cli.py]: Agreed. Missed that.

**Comment by stdavis** [comment]
LGTM

**Pull request merged**